docs/workflows/copy.rst

```diff
@@ -47,47 +47,62 @@ depend on the existence of other content units to be "correct" or meaningful. Fo
+    are necessary for the copy operation, **it will continue to execute the copy**. This
+    can happen for a variety of reasons (e.g., the source repositor(ies) don't contain all
+    of the referenced/required content, or the copy-configuration is incomplete or otherwise
+    incorrect). In these instances, missing entities are logged as WARNINGs - be sure to
+    check the logs if the results of a copy are unexpected.
 
 Dependency solving
 __________________
 
-When copying RPM packages between repositories, it is likely desirable to ensure that all of the
-RPM dependencies they need to be "installable" on a client system are also present in the
-repository they are being copied to. For example, if you want to add the "hexchat" RPM to a
-new repository, and you want to be able to install it from that repository, the repository should
-also contain its dependencies such as "libnotify" and "gtk3". This applies likewise to all RPM
-concepts that "contain" RPMs also, such as Advisories (Errata), Modules, and PackageGroups. With
-the RPM copy API, you are afforded the option to have all dependencies (and the dependencies of
-those dependencies) copied for you automatically in the background, if they do not already exist
-in the destination repository. In Pulp 2, this feature was called "recursive copy" because in a
-sense it copies dependencies recursively.
+When copying RPM packages, advisories, or modules between repositories, it is useful to ensure
+that all of the RPM dependencies they need to be "installable" on a client system are also present
+in the destination-repository.
+
+For example: if you want to add the "hexchat" RPM to a new repository, and you want to be able
+to install it from that repository, the repository should also contain hexchat's dependencies
+such as "libnotify" and "gtk3".
+
+This applies to all RPM concepts that "contain" RPMs, such as Advisories (Errata), Modules, and
+PackageGroups. With the RPM copy API, you are afforded the option to have all dependencies (and
+the dependencies of those dependencies) copied for you automatically, if they do not already exist
+in the destination repository. In Pulp 2, this feature was documented as "advanced copy";
+the Pulp3 behavior matches the "recursive-conservative" option from Pulp2 (copies **latest missing**
+dependencies)
 
 Unfortunately, solving these complex dependency relationships can be quite expensive. For this
-reason, the functionality to perform these checks is not enabled by default, and is instead
+reason, the functionality to perform these checks is **not enabled by default**, and is instead
 provided by setting the "dependency_solving" parameter to a value of True when making calls against
-the API. Note that if you do elect to not perform dependency solving, it will be possible to
-accidentally create incomplete repositories, if you are not careful.
+the API. Note that if you do not choose dependency-solving, it is possible to create incomplete
```

Comment:
perhaps a reminder about incomplete configuration

and maybe we should call out baseos and appstream specifically as a case where you need to be careful

Reply:
See the note at 73 - do we need to restate here? And we call out Base/Appstream explicitly starting at 118, is that sufficient?

Reply:
Not at length, but maybe just 

> if you do not choose dependency-solving (or configure it incorrectly), it is possible to create incomplete repositories

I missed the appstream / baseos mention though since it was collapsed - you're right, we don't need to add anything more.